css/properties/print-color-adjust.json

```diff
@@ -1,55 +1,67 @@
 {
   "css": {
     "properties": {
-      "color-adjust": {
+      "print-color-adjust": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color-adjust",
-          "spec_url": "https://drafts.csswg.org/css-color-adjust/#perf",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/print-color-adjust",
+          "spec_url": "https://drafts.csswg.org/css-color-adjust/#propdef-print-color-adjust",
           "support": {
             "chrome": {
               "version_added": "49",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "chrome_android": {
               "version_added": "49",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "edge": {
               "version_added": "79",
-              "alternative_name": "-webkit-print-color-adjust"
-            },
-            "firefox": {
-              "version_added": "48"
-            },
-            "firefox_android": {
-              "version_added": "48"
+              "prefix": "-webkit-"
             },
+            "firefox": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "48",
+                "alternative_name": "color-adjust"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "48",
+                "alternative_name": "color-adjust"
```

Comment:
@ddbeck There is no formal way to indicate that this version is deprecated. Should we/do we need a note?:

> `color-adjust` is a deprecated shorthand version of the property name.

Reply:
We're papering over the spec authors doing something unusual and confusing here. If `color-adjust` is ever used for something independently of `print-color-adjust`, we'll make a separate feature (and page) and we can address it then. But as it is, I don't think we need to acknowledge the status of `color-adjust`.